blog/2024-04-05-pi4micronaut.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Pi4Micronaut Presentation
+description: Pi4Micronaut, a Java library by OpenSourcewithSLU, enables development on Raspberry Pi. Version 1.0 was released in the Maven Cental repository on Pi Day.
+slug: blog_presentation_pi4micronaut
+authors: [Yash Kamal Bhatia]
+tags: [announcement, news]
+image: ./pi4micronaut-20240314/Baking_Pi4Micronaut.png
+hide_table_of_contents: false
+---
+
+Ruthvik presented the Pi4Micronaut. Students got a hands-on practice using the pi4micronaut library. It was a great success that exceeded the expectations of all the planners. 
+
+<!--truncate-->
+
+**What:** Pi4Micronaut <br/>
+**Who:** Presented by [Ruthvik Mannem](https://github.com/ruthvikm), MS student in Artificial Intelligence<br/>
+**When:** Friday, April 5<br/>
+  2:00-3:00 p.m. (1 hour)<br/>
+**Where:** [Pius Library, ATC](https://www.slu.edu/library/services/academic-technology-commons/index.php)<br/>
+**Resources:** [presentation of instructions](https://oss-slu.github.io/Pi4Micronaut/)
+
+A group of students and staff huddled over laptops around a large conference table in the Academic Technology Commons of Saint Louis University's Pius Library. On an already busy Friday afternoon these students had elected to spend their time getting some hands on experiences with open source software. 
+
+Ruthvik Mannem, a second year Masters student in Artificial Intelligence and a Research Assistant with OpenSourcewithSLU, Explained about how he had created a library for micronaut which worked on Java instead of Python. Then he went [step-by-step](https://docs.google.com/presentation/d/1etn4jDqLUA8tdQhA8emUGaOXri8pRJKu/edit?usp=sharing&ouid=107304909661279551530&rtpof=true&sd=true) with students down the path of installing the libraries and packages to communicate with the micronaut  ![Raspberry Pi 4](./oss-atc-20240405/20240405_3.jpeg). The Library works upto Raspberry Pi 4B  Ruthvik then walked those attending the workshop through actually using the LED/LCD Functions along with the Bulb Functionality they created as a [Example](./oss-atc-20240405/20240405_4.jpeg). 
```

Comment:
Micronaut should be capitalized. I think this maybe should read "to communicate with the Micronaut library." (end with "library" and put the period before the inline image.) 

"up to"

"as an [example]"

Reply:
maybe "Micronaut framework" instead :D